src/components/MainMap.tsx

```diff
@@ -98,14 +99,38 @@ export const MainMap = ({
         <div ref={geocoderContainerRef} className="z-1 fixed top-2 right-12" />
         <Source type="geojson" data={sidewalkFeatureCollection}>
           <Layer
-            id="Data"
+            id="Paths"
             type="line"
+            filter={[
+              "match",
+                ["get", "highway"],
+                paths,
+                true,
+                false
+              ]} // Filter for true paths
             paint={{
               "line-color": "red",
               "line-opacity": 0.8,
               "line-width": 3,
             }}
           />
+          <Layer
+            id="Roads"
+            type="line"
+            filter={
+            ["match",
+                ["get", "highway"],
+                paths,
+                false,
+                true
+            ]} // Filter for non-paths (roads)
+            paint={{
+              "line-color": "red",
+              "line-opacity": 0.8,
+              "line-width": 2,
+              "line-dasharray": [1, 1 ],
```

Comment:
Nit
```suggestion
              "line-dasharray": [1, 1],
```

Reply:
good spot there. I'll look for a config for the linter that spots this as well.